README.adoc

```diff
@@ -131,6 +131,24 @@ This can be achived with the {uri-kube-apiserver}[Kubernetes API server CLI], e.
 $ kube-apiserver --cors-allowed-origins .*
 ```
 
+== Authentication
+
+We try to support the various authentication strategies supported by https://kubernetes.io/docs/reference/kubectl/overview/[kubectl] in order to provide seamless integration with your local setup. Here are the different authentication strategies we support depending on how you're using Kubebox:
+
+[cols="<,^,^,^,^,^", options="header"]
+|===
+||Executable |Server |Kubectl | Docker| Online
+
+|OpenID Connect | icon:check[] | icon:check[] | icon:check[] | icon:check[] | icon:check[]
+
+|Exec (Amazon EKS, Digital Ocean) | icon:check[] | | | |
```

Comment:
I would not mention `Exec`, which is rather an implementation details, and add lines for EKS and DO.